src/boot/api.js

```diff
@@ -16,14 +17,27 @@ const signTransaction = async function (actions) {
   let error = null
   try {
     if (this.$type === 'ual') {
+      let options = {}
+      if (Screen.gt.md) {
+        options = {
+          blocksBehind: 3,
+          expireSeconds: 30,
+          text: 'Scan the QR-code with Hypha Wallet on your mobile device in order to sign this transaction request',
```

Comment:
@evgenibir this text should be translated

Reply:
fixed